hack/testing-olm/test-657-im-block-autocreate-for-write-suffix.sh

```diff
@@ -53,40 +53,7 @@ if [ "${DO_SETUP:-true}" == "true" ] ; then
-  os::cmd::expect_success "${repo_dir}/hack/deploy-example-secrets.sh  ${TEST_NAMESPACE}"
-  os::cmd::expect_success "oc -n ${TEST_NAMESPACE} create -f ${repo_dir}/hack/cr.yaml"
-
-  os::log::info "---------------------------------------------------------------"
-  os::log::info "Waiting for elasticsearch-operator to deploy the cluster..."
-  os::log::info "---------------------------------------------------------------"
-  os::cmd::try_until_success "oc -n ${TEST_NAMESPACE} get deployment -l component=elasticsearch -o jsonpath='{.items[0].metadata.name}'" $((2 * $minute))
-
 fi
-#wait for pod
-os::log::info "---------------------------------------------------------------"
-os::log::info "Waiting for elasticsearch deployment to be ready..."
-os::log::info "---------------------------------------------------------------"
-wait_for_deployment_to_be_ready ${TEST_NAMESPACE} $(oc -n ${TEST_NAMESPACE} get deployment -l component=elasticsearch -o jsonpath={.items[0].metadata.name}) $((2 * $minute))
-
-pod=$(oc -n $TEST_NAMESPACE get pod -l component=elasticsearch -o jsonpath={.items[0].metadata.name})
-os::log::info "---------------------------------------------------------------"
-os::log::info Attempt to autocreate an index without a '-write' suffix...
-os::log::info "---------------------------------------------------------------"
-os::cmd::expect_success_and_text "oc -n $TEST_NAMESPACE exec $pod -c elasticsearch -- es_util --query=foo/_doc/1 -d '{\"key\":\"value\"}' -XPUT -w %{http_code}" ".*201"
-
-os::log::info "---------------------------------------------------------------"
-os::log::info Attempt to autocreate an index with a '-write' suffix...
-os::log::info "---------------------------------------------------------------"
-os::cmd::expect_success_and_text "oc -n $TEST_NAMESPACE exec $pod -c elasticsearch -- es_util --query=foo-write/_doc/1 -d '{\"key\":\"value\"}' -XPUT -w %{http_code}" ".*404"
-
-os::log::info "---------------------------------------------------------------"
-os::log::info Explicitly creating an index with a '-write' suffix...
-os::log::info "---------------------------------------------------------------"
-os::cmd::expect_success_and_text "oc -n $TEST_NAMESPACE exec $pod -c elasticsearch -- es_util --query=foo-write -XPUT -w %{http_code}" ".*200"
 
-os::log::info "---------------------------------------------------------------"
-os::log::info Verifying can write to index with a '-write' suffix...
-os::log::info "---------------------------------------------------------------"
-os::cmd::expect_success_and_text "oc -n $TEST_NAMESPACE exec $pod -c elasticsearch -- es_util --query=foo-write/_doc/1 -d '{\"key\":\"value\"}' -XPUT  -w %{http_code}" ".*201"
+TEST_WATCH_NAMESPACE=${TEST_NAMESPACE} TEST_OPERATOR_NAMESPACE=${TEST_NAMESPACE} \
+  go test -v ./test/e2e-olm/... -kubeconfig=${KUBECONFIG} -parallel=1 -timeout 1500s -run TestElasticsearchWrite
```

Comment:
Missing passing this test to the junit report like [here](https://github.com/openshift/elasticsearch-operator/pull/643/files#diff-fa5e2f28a843bccb27a13c4417f9866a2cc257327be992150e11a92408ff0e59R56)

Reply:
Fixe. Thanks!